### PR TITLE
major microservice updates

### PR DESCRIPTION
**Pull request by scarney81**
will be major rev 5.0.0

opt-in request logging
opt-in create server
eslint -> standard

### CODE REVIEW

File: create-app.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+var _ = require('lodash')
```
Comment (scarney81) [inline create-app.js]: This file is a rename of `server.js` with changes to pass lint.

File: create-server.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const { createServer } = require('http')
```
Comment (scarney81) [inline create-server.js]: This is a net new file

File: middleware/log.js [hunk 3]
```diff
@@ -1,34 +1,32 @@
...
     }
 
     if (user) {
-      response.push('org: ' + user.organization_id);
-      response.push('plan: ' + user.organization_plan);
-      response.push('user: ' + user.id);
+      response.push('org: ' + user.organization_id)
+      response.push('plan: ' + user.organization_plan)
+      response.push('user: ' + user.id)
     }
```
Comment (DannyDouglass) [inline middleware/log.js]: might want to include `organization_trim` here

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lanetix/microservice",
-  "version": "4.1.1",
+  "version": "5.0.0-next.21",
```
Comment (tone81) [inline package.json]: Wat dis?

Comment (scarney81) [reply package.json]: I'm publishing these with a tag called `next`. This is versioning to support that.

**Comment by scarney81** [comment]
@tone81 - technically, this module already has a `node 8.9.4` dependency if you check the `package.json`

File: middleware/log.js (hunk 3 above)
Comment (scarney81) [reply middleware/log.js]: _Might_ want to. This file didn't get any changes outside of passing lint. Do we guarantee it's on the JWT?

Comment (tone81) [reply middleware/log.js]: Yup!

**Comment by tone81** [comment]
Oh, that Jonathan. :shakesfist:

**Pull request merged**